virttest/qemu_vm.py

```diff
@@ -896,25 +896,35 @@ def add_memorys(devices, params):
 
             :param devices: VM devices container
             """
-            options = []
+            options, devs = [], []
+            params.setdefault("mem", "512")
             mem_params = params.object_params("mem")
-            if mem_params.get("maxmem"):
+            mem_params.setdefault("automem", "yes")
```

Comment:
@xutian you have mentioned migration will not work if `automem` is `yes`, so should we have a check here if a migration test uses to add memory controller, then we have to  `mem_params.setdefault("automem", "no")`